subprojects/diagnostics/src/main/java/org/gradle/api/reporting/dependencies/HtmlDependencyReportTask.java

```diff
@@ -176,6 +178,7 @@ public Set<Project> getProjects() {
      *
      * @param projects The set of projects. Must not be null.
      */
+    @Override
```

Comment:
Why override these things instead of reusing the parent implementation?

Reply:
I didn't look into doing that, I just added the missing annotation here and preserved the behavior.

Reply:
Just rebased and got to these.  If it passes testing, I'll merge it for 8.10.